cmake/Modules/Findcpp_redis.cmake

```diff
@@ -0,0 +1,55 @@
+add_library(cpp_redis UNKNOWN IMPORTED)
+
+find_path(cpp_redis_INCLUDE_DIR cpp_redis/cpp_redis)
+mark_as_advanced(cpp_redis_INCLUDE_DIR)
```

Comment:
https://cmake.org/cmake/help/v3.0/command/mark_as_advanced.html
>Mark the named cached variables as advanced. An advanced variable will not be displayed in any of the cmake GUIs unless the show advanced option is on.

Not sure if this is required, but it was used in examples of `Find*.cmake` files